airbyte-webapp/src/components/ui/Toast/Toast.module.scss

```diff
@@ -43,6 +43,7 @@ $toast-bottom-margin: 27px;
   position: fixed;
   box-sizing: border-box;
   bottom: $toast-bottom-margin;
+  margin-left: calc(vars.$width-size-menu / 2);
```

Comment:
This variable name doesn't make much sense to me as we aren't using it anywhere for a menu, and I can't tell why a menu's width would be a relevant benchmark for this.  

Is there a meaningful reason to use 2 * width-size-menu vs. the number itself?

Reply:
Agreed, it's a weird naming choice. Here, I used that variable purely because the modal does, to keep the two nicely-aligned vis a vis each other.

Zooming out a little, it's not used very many places:
![2023-01-20-13:31:07-screenshot](https://user-images.githubusercontent.com/7516653/213808941-1b752d32-ea45-473f-8202-08e922198f3a.png)
I think it's a fine candidate to be pulled out or renamed or something, I just didn't want to have to verify the correctness of `components/connection/CatalogTree/next/StreamDetailsPanel` in order to ship the free connectors program :grimacing: